sensor_msgs/Image image
---
string obj